packages/@lightningjs/ui-components/package.json

```diff
@@ -27,12 +27,14 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --no-coverage --watch --colors"
   },
   "peerDependencies": {
-    "@lightningjs/core": "2.11.0"
+    "@lightningjs/core": "^2.7.0"
```

Comment:
What does this need to be rolled back for? It seems like it's still resolving to the same version as before, and I think we needed to keep this locked at 2.11.0 until we looked more into the issues related to #400.

Reply:
Oh it shouldn't be. Will update, thanks!

Reply:
What if we did something like this?

`"@lightningjs/core": ">=2.0.0 <=2.11.0"`

https://docs.npmjs.com/cli/v10/commands/npm-install

That way the consuming application does not have to be locked to a specific version but does not hit the version that has breaking changes.